content/src/content/jcr_root/apps/core/wcm/components/title/v2/title/title.html

```diff
@@ -19,9 +19,10 @@
      data-sly-test.text="${title.text}"
      data-cmp-data-layer="${title.data.json}"
      id="${component.id}"
-     class="cmp-title">
+    class="cmp-title">
     <h1 class="cmp-title__text" data-sly-element="${title.type}"><a
-            data-sly-unwrap="${!title.linkURL || title.linkDisabled}" class="cmp-title__link" href="${title.linkURL}"
+            data-sly-unwrap="${!title.link.htmlAttributes['href'] || title.linkDisabled}" class="cmp-title__link"
```

Comment:
`title.link` cannot be used in v1 (see comment above)

Reply:
but this is v2, am I missing something?

Reply:
@jckautzmann is right, we should keep the old `title.linkURL` for `href` (for backwards compatibility) and only use `title.link.htmlAttributes` for the two new attributes. It will not output any errors, just skip the attributes if `title.link` is not defined/null.